packages/functionals/botpress-qna/src/middleware.js

```diff
@@ -27,7 +27,7 @@ export const processEvent = async (event, { bp, storage, logger, config }) => {
 
   if (answer.action.includes('text')) {
     logger.debug('QnA: replying to recognized question with plain text answer', answer.id)
-    event.reply(config.textRenderer, { text: answer.answer })
+    event.reply(config.textRenderer, { text: answer.answer, typing: '1s' })
```

Comment:
@epaminond how do you want this to be an option ? it could be something like each q&r has a typing time defined and it can be edited in the create question form ?

Reply:
@xavierbriole , for now I'd suggest to make `typing` param come from env-variable defaulting to `false`.